### ISSUE TITLE
Error with puppet-retrspec with version 0.4.0

### ISSUE THREAD

**Issue by petems** [nwops/puppet-retrospec#33]
```
22:07:38 petersouter@Peters-MacBook-Pro-3 ~/projects/petems-hyperic_agent (master #%) $ retrospec -v
WARN: Unresolved specs during Gem::Specification.reset:
      CFPropertyList (~> 2.2.6)
WARN: Clearing out unresolved specs.
Please report a bug if this causes problems.
0.4.0 (c) Corey Osman
22:07:40 petersouter@Peters-MacBook-Pro-3 ~/projects/petems-hyperic_agent (master #%) $ retrospec puppet -v
WARN: Unresolved specs during Gem::Specification.reset:
      CFPropertyList (~> 2.2.6)
WARN: Clearing out unresolved specs.
Please report a bug if this causes problems.
/opt/rubies/2.1.2/lib/ruby/gems/2.1.0/gems/retrospec-0.4.0/lib/retrospec/plugins/v1/plugin.rb:43:in `run_cli': uninitialized constant Retrospec::Plugins::V1::Plugin::NotImplemented (NameError)
	from /opt/rubies/2.1.2/lib/ruby/gems/2.1.0/gems/retrospec-0.4.0/lib/retrospec/cli.rb:39:in `run'
	from /opt/rubies/2.1.2/lib/ruby/gems/2.1.0/gems/retrospec-0.4.0/bin/retrospec:5:in `<top (required)>'
	from /opt/boxen/rbenv/versions/2.1.2/bin/retrospec:23:in `load'
	from /opt/boxen/rbenv/versions/2.1.2/bin/retrospec:23:in `<main>'
```

**Comment by logicminds** [comment]
I had to change the api around in 0.4.0 of retrospec.  I haven't had a chance to update puppet-retrospec yet but the the gem should have been pinned to version 0.3.0 anyways.   Did you install retrospec 0.4.0 by itself?  Until I release the updated puppet-retrospec version you will have to use retrospec v0.3.0 when using the puppet plugin.  I expect to get this new version out tonight.

**Comment by petems** [comment]
Ok, no worries, I'll update when the new one is out :+1:

**Comment by logicminds** [comment]
Ok, I just tried this myself and for some reason the gem spec is not enforcing the gem version.

**Comment by logicminds** [comment]
fixed in version 0.10.0
